manifests/init.pp

```diff
@@ -0,0 +1,48 @@
+# @summary Profile for installing and managing Java.
+#
+# Profile for installing and managing Java.
+#
+# @param install_lock_files
+#   Boolean - Install lock files or not.
+#
+# @param lock_files
+#   Hash of lock files (and parent dir(s)).
+#
+# @param pkg_list
+#   Hash of package names to install
+#   Keys must be an OS Family name
+#   Values must be a Hash of the format:
+#       Key = package name
+#           A package name can be prefixed with "-" to remove it from the list.
+#           In this way, a higher level hiera layer can remove a package added
+#           by a lower, common layer.
+#
+#       Value (optional):
+#           1. <EMPTY> in which case, the package will be installed
+#           2. <HASH> conataining valid attributes for the 
+#              "package" native puppet resource type
+#
+# @example
+#   include profile_java
+class profile_java (
+  Boolean $install_lock_files,
+  Hash    $lock_files,
+  Hash    $pkg_list,
+) {
+  # Install packages
+  $pkg_defaults = {
+    ensure => 'installed',
+  }
```

Comment:
I do not think this is being referenced anywhere. Either reference it as a default parameter in line #36, or convert it to just be:

```suggestion
  Package [
    ensure => 'installed',
  ]
```

Reply:
Fixed.